postreise/plot/plot_carbon_bar.py

```diff
@@ -1,51 +1,73 @@
 import numpy as np
 from matplotlib import pyplot as plt
+from powersimdata.scenario.scenario import Scenario
 
+from postreise.analyze.generation.emissions import (
+    generate_emissions_stats,
+    summarize_emissions_by_bus,
+)
 
-def plot_carbon_bar(carbon_by_bus_1, carbon_by_bus_2, scenario_name_1, scenario_name_2):
-    """Makes bar chart of carbon emissions by fuel type for two scenarios.
 
-    :param dict carbon_by_bus_1: emission by type and bus for first scenario
-        as returned by :func:`postreise.analyze.carbon.summarize_carbon_by_bus`.
-    :param dict carbon_by_bus_2: emission by type and bus, for second scenario
-        as returned by :func:`postreise.analyze.carbon.summarize_carbon_by_bus`.
-    :param str scenario_name_1: name of first scenario, for labeling plot.
-    :param str scenario_name_2: name of second scenario, for labeling plot.
+def plot_carbon_bar(*args, name=None):
```

Comment:
What do you think about `name` -> `labels`? Seems more precise

Reply:
Done